charts/alfresco-common/Chart.yaml

```
@@ -5,9 +5,5 @@ description: |
   A helper subchart to avoid duplication in alfresco charts and set common
   external dependencies
 type: library
-version: 3.1.4
-dependencies:
-  - name: common
-    repository: oci://registry-1.docker.io/bitnamicharts
-    version: 1.x.x
+version: 4.0.0
```

```suggestion
version: 4.0.0-alpha.0
```

given that we need to test every other chart before